src/app/Main/Body/QueryPage/Drawer/Displayed_Features/DisplFeatures-Selector.component.tsx

```diff
@@ -105,11 +101,11 @@ export function DisplayedFeatureSelectorComponent(
             label={props.label}
             noOptionLabel={t("Drawer.Selector")}
             dropDownValuesObj={dropDownValuesObj}
-            selectedValuesObj={selectedValueObj}
+            selectedValuesObj={selectedValueObjList}

```

Comment:
Here you would convert your internal data to the required one. E.g.:
`{selectedValueObj === null ? [] : [selectedValueObj]}`

Reply:
Done.